modules/dtj/Animal/scripts/racehorse.pl

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+use FindBin;
+use lib "$FindBin::Bin/../lib";
+
+use RaceHorse;
+
+my @horses;
+push @horses, RaceHorse->named('Billy Boy');
+push @horses, RaceHorse->named('Furious D');
+push @horses, RaceHorse->named('My Little Pony');
+
+$horses[0]->won;
+$horses[1]->lost;
+$horses[2]->lost;
+
+foreach my $horse ( @horses ) {
+  print $horse->name, ' has standings ', $horse->standings, ".\n";
+}
```

Comment:
Would be nice to see some racehorses going out of context, and showing that the files are created at this point. Also that the data is reloaded upon creating a racehorse with the same name.